Tasks/AzureRmWebAppDeploymentV4/operations/TaskParameters.ts

```diff
@@ -119,6 +119,7 @@ export class TaskParametersUtility {
         taskParameters.PublishProfilePath = tl.getInput('PublishProfilePath', true);
         taskParameters.PublishProfilePassword = tl.getInput('PublishProfilePassword', true);
         taskParameters.Package = new Package(tl.getPathInput('Package', true));
+        taskParameters.SetParametersFile = tl.getPathInput('SetParametersFile', false);
```

Comment:
Don't we need this input on UI? this will work with yaml based pipeline only right?

Reply:
Yes correct. Will update the visibility rule for input UI.